test/Microsoft.TestTemplates.AcceptanceTests/DotnetCoreTemplateTests.cs

```diff
@@ -29,6 +29,9 @@ public class DotnetCoreTemplateTests : AcceptanceTestBase
             "MSTest-CSharp",
             "MSTest-FSharp",
             "MSTest-VisualBasic",
+            "NUnit-CSharp",
```

Comment:
Is there some way we could validate Item templates as well?

Reply:
@AbhitejJohn , contents of the templates were changed a little (template.json mostly). It were divided to 2.1, 3.0, 3.1 and 5.0 packages (in https://github.com/nunit/dotnet-new-nunit all templates were in one nuget package).
Important changes: `groupIdentity` and `identity` has been changed. 

Therefore I assume that we need to replace bundled nunit templates in dotnet/installer (like https://github.com/dotnet/installer/pull/7169) with Microsoft.DotNet.Test.ProjectTemplates.3.0 and remove reference to https://www.nuget.org/packages/NUnit3.DotNetNew.Template/ package from installer.

Reply:
Yes, we could 

- create test project 
- do `dotnet new nunit-test -n some-new-test-class`,
- run `dotnet test` on the test project
- check that tests are passed successfully and count of tests is equal to 2.

Reply:
Do you want to implement that as part of this PR?

Reply:
I've just found some time to implement item templates tests. Please, review my last commit.

Unfortunately, CI checks are failed on `Build And Publish` phase with following error:

```
The command ""F:\workspace\_work\1\s\.packages\microsoft.diasymreader.pdb2pdb\1.1.0-beta2-19575-01\tools\Pdb2Pdb.exe" "F:\workspace\_work\1\s\artifacts\bin\Microsoft.TestTemplates.AcceptanceTests\Release\netcoreapp3.1\Microsoft.TestTemplates.AcceptanceTests.dll" /out "F:\workspace\_work\1\s\artifacts\SymStore\Release\Microsoft.TestTemplates.AcceptanceTests\netcoreapp3.1\Microsoft.TestTemplates.AcceptanceTests.pdb" /srcsvrvar SRC_INDEX=public" exited with code -1.
```

Same error exists in other PRs like in [`Test net5.0 templates`](https://dev.azure.com/dnceng/public/_build/results?buildId=732973&view=results).

Is build broken?

Reply:
Seems like it, I am investigating it, it is probably a dependency issue because it started failing even on stuff that worked before.

Reply:
I fixed the build and the item templates, for F# it was not detecting your new file so we got only 1 test result. This is because F# project do not include all files by default, instead you need the explicitly list them in the project in the right order to avoid crazy dependency graphs. I could not find any commandline option, probably because you rarely know where exactly the file needs to end up. So I just modify the file with a bit of string matching.

Reply:
Thanks for fixing tests for vb names and f# files! I can't imagine why I've missed that though.